auto_submit/lib/requests/github_webhook.dart

```diff
@@ -25,19 +33,432 @@ class GithubWebhook {
     final String rawBody = await request.readAsString();
     final body = json.decode(rawBody) as Map<String, dynamic>;
 
-    if (!body.containsKey('pull_request') || !body['pull_request'].containsKey('labels')) {
+    if (!body.containsKey('pull_request') ||
+        !body['pull_request'].containsKey('labels')) {
       return Response.ok(jsonEncode(<String, String>{}));
     }
 
     PullRequest pullRequest = PullRequest.fromJson(body['pull_request']);
-    hasAutosubmit = pullRequest.labels!.any((label) => label.name == 'autosubmit');
+    hasAutosubmit =
+        pullRequest.labels!.any((label) => label.name == 'autosubmit');
 
     if (hasAutosubmit) {
-      // TODO(kristinbi): Check if PR can be submitted. https://github.com/flutter/flutter/issues/98707
+      // TODO(kristin): hardcode the githubToken this time for test, will add it to env later.
+      // final String githubToken = Platform.environment['AUTOSUBMIT_TOKEN']!;
+
+      final String githubToken = 'f4d8bc081a5f3ad57f3df3a99ec0417b269cec90';
```

Comment:
You don't want to submit with the real token. I believe this is for local test purpose, and for this PR I suggest bring line 47 back.